assets/html/custombar.html

```diff
@@ -7,18 +21,33 @@
       <div class="bounce3"></div>
     </div>
   </div>
+  <label class="letter-label">
+        <input class="js-bag-input letters-input" type="text" maxLength="2" pattern="[A-Z]" placeholder="A-Z"/>
+  </label>
+  <div class="tab-cnr">
+    <div class="tab-buffer">
```

Comment:
if these tab-buffers are just for spacing, you should use css. seems like you'd be able to accomplish the same thing with:

`.tab-cnr { padding-left: 35%; padding-right: 35% }`

in general you should never need to resort to using a DOM node for spacing. In some cases you need a container, but you want to keep the html minimal and rely on css wherever possible.